### ISSUE TITLE
Potential bug: missing axis parameter

### ISSUE THREAD

**Issue by mle-els** [DSE-MSU/DeepRobust#70]
The `argmax` call might need `axis=1` to select the largest logit per example:

        secondlargest = np.argmax((logits.double() * secondlargest_mask).cpu().detach().numpy())

https://github.com/DSE-MSU/DeepRobust/blob/master/deeprobust/image/attack/cw.py#L209
